src/node.js

```diff
@@ -73,6 +76,30 @@
         NativeModule.require('_third_party_main');
       });
 
+    } else if (process.openManPage) {
+      // Attempt to spawn `man 1 node`.
+      const cp = NativeModule.require('child_process');
```

Comment:
Why not start now?  If nothing else, the more code exists in the main startup function, the slower code generation (and thus startup time) becomes.